community/libsoup/build

```diff
@@ -11,6 +11,7 @@ meson \
     -Dgssapi=disabled \
     -Dbrotli=disabled \
     -Dsysprof=disabled \
+	-Dpkcs11_tests=disabled \
```

Comment:
Would you happen to be using tabs here instead of spaces?

Reply:
Oops, fixed in ba37fca